### PR TITLE
Fix old nft storage

### PR DESCRIPTION
**Pull request by iulianpascalau**
## Reasoning behind the pull request
- moving old-type NFTs across shards while changing their metadata encountered issues by having different metadata versions in each shard
  
## Proposed changes
- update each time the NFT metadata whenever the need for processing them. This is the integration branch for the fix PR found here https://github.com/multiversx/mx-chain-vm-common-go/pull/164
- refactored activation flags as they will be enabled on the mainnet
- changed the blockchain hook counters according to the last tests

## Testing procedure
- standard system test
- manipulate old-type NFTs while changing their metadata fields. With the fix activated, the changes should propagate across shards each time.

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

File: common/enablers/enableEpochsHandler.go [hunk 1]
```diff
@@ -116,6 +116,7 @@ func (handler *enableEpochsHandler) EpochConfirmed(epoch uint32, _ uint64) {
 	handler.setFlagValue(epoch >= handler.enableEpochsConfig.RuntimeMemStoreLimitEnableEpoch, handler.runtimeMemStoreLimitFlag, "runtimeMemStoreLimitFlag")
 	handler.setFlagValue(epoch >= handler.enableEpochsConfig.MaxBlockchainHookCountersEnableEpoch, handler.maxBlockchainHookCountersFlag, "maxBlockchainHookCountersFlag")
 	handler.setFlagValue(epoch >= handler.enableEpochsConfig.WipeSingleNFTLiquidityDecreaseEnableEpoch, handler.wipeSingleNFTLiquidityDecreaseFlag, "wipeSingleNFTLiquidityDecreaseFlag")
+	handler.setFlagValue(epoch > handler.enableEpochsConfig.AlwaysSaveTokenMetaDataEnableEpoch, handler.alwaysSaveTokenMetaDataFlag, "alwaysSaveTokenMetaDataFlag")
```
Comment (sasurobert) [inline common/enablers/enableEpochsHandler.go:119]: >=

Comment (sasurobert) [reply common/enablers/enableEpochsHandler.go:119]: ">="

Comment (iulianpascalau) [reply common/enablers/enableEpochsHandler.go]: right, fixed

[the remark this answers is not in the document]
Comment (iulianpascalau) [reply common/enablers/enableEpochsHandler.go]: >= is correct, fixed

**Review by gabi-vuls — DISMISSED** [review]
System test passed

**Review by sasurobert — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]

**Review by SebastianMarian — APPROVED** [review]

**Pull request merged**